src/viking.js

```diff
@@ -1,11 +1,89 @@
 // Soldier
-function Soldier() {}
+function Soldier(health, strength) {
+  this.health = health;
+  this.strength = strength;
+  this.attack = function () {
```

Comment:
`this.attack` and `this.reciveDamage` are methods and the go in the `prototype` object which means you have to use the next syntax: 
```
Soldier.prototype.attack = function () => {
  return this.strength;
}
```